### ISSUE TITLE
[Chromium] Screen reader does not identify toggle buttons as toggles on welcome page

### ISSUE THREAD

**Issue by acornestean** [mozilla-extensions/firefox-bridge#56]
**Description:**
Screen reader does not identify the link handling and telemetry opt in/out toggles as toggles. Only the strings are announced.

**Preconditions:**
1. A screen reader is enabled in the OS

**Steps to reproduce:**
1. Load the Chromium Firefox Bridge v0.1 in the browser
2. The welcome page opens in a new tab and the screen reader start announcing the page content
3. Notice the screen reader does not announce the toggle buttons as toggle and only reads the associated strings

**Expected:**
The screen reader should announce the toggle buttons as toggles.

**Actual:**
The screen reader does not announce the toggle buttons as toggles and only reads the associated strings.

**Comment by acornestean** [comment]
Verified as Fixed with Alpha Build V2 extension.

Screen reader identifies toggles as “checkboxes” stating whether they’re checked or unchecked.
